templates/msgext/js/default/index.js

```diff
@@ -40,7 +40,7 @@ const messageExtensionBot = new MessageExtensionBot();
 // Create HTTP server.
 const server = express();
 const port = process.env.port || process.env.PORT || 3978;
-server.listen(port, () => console.log(`\Bot/ME service listening at https://localhost:${port}`));
+server.listen(port, () => console.log(`\Bot/Message Extension service listening at https://localhost:${port}`));
```

Comment:
Would `Messaging extension` be better? refer to official docs like https://docs.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/what-are-messaging-extensions

Reply:
Ok, thanks, will update